src/entry-point.tsx

```diff
@@ -4,6 +4,8 @@ import * as ReactDOM from "react-dom";
 import { LibraryContainer } from "./components/LibraryContainer";
 import { JsonDownloader } from "./LibraryUtilities";
 import { Reactor, Event } from "./EventHandler";
+import "core-js";
```

Comment:
Hi @mjkkirschner I did not understand the purpose from your commit message, can you elaborate?

Reply:
core-js includes a bunch of polyfills - so that browsers (like ie) which are missing some functions can use them - in this case I needed `string.prototype.startsWith` but we could probably just add the polyfill for that single method.... this just works though.